cmd/support.go

```diff
@@ -2,17 +2,81 @@ package cmd
 
 import (
 	"fmt"
+	"net/url"
 
+	"github.com/manifoldco/promptui"
 	"github.com/spf13/cobra"
 )
 
+const (
+	baseURL              = "https://connect.dev.redhat.com/support/technology-partner/#/case/new?"
+	typeParam            = "type"
+	typeValue            = "CERT"
+	sourceParam          = "source"
+	sourceValue          = "preflight"
+	certProjectTypeParam = "cert_project_type"
+	certProjectIDParam   = "cert_project_id"
+	pullRequestURLParam  = "pull_request_url"
+)
+
 var supportCmd = &cobra.Command{
 	Use:   "support",
 	Short: "Submits a support request",
-	Long: `This command will submit a support request to Red Hat along with the logs from the latest Preflight checck.
+	Long: `This command will submit a support request to Red Hat along with the logs from the latest Preflight check.
 	This command can be used when you'd like assistance from Red Hat Support when attempting to pass your certification checks. `,
```

Comment:
We need to reword this. This command now returns a "fast-track" way to create a support request, but doesn't actually submit the support request because it requires authentication, correct?

Reply:
You can word this however you think makes sense, but the current wording doesn't seem right to me.

Reply:
You are correct, this was the existing skeleton code, but because of sso login requirement this will only print out a URL for the user to copy/paste into a browser and then login to be able to submit. I'll think about a more accurate wording.

Reply:
This has been updated as well.